x/liquidstakeibc/migrations/v3/store.go

```diff
@@ -0,0 +1,47 @@
+package v3
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/persistenceOne/pstake-native/v2/x/liquidstakeibc/types"
+)
+
+// MigrateStore performs in-place store migrations from v2.3.0 to v2.
+// The migration includes:
+//
+// - Migrate host chains to include the MaxEntries and AcceptableDelta attribute.
+func MigrateStore(ctx sdk.Context, storeKey storetypes.StoreKey, cdc codec.BinaryCodec) error {
+
+	for _, hc := range getAllHostChains(ctx, storeKey, cdc) {
+		hc.Params.MaxEntries = 7
```

Comment:
yes, it will.. 
https://rest.osmosis-1.audit.one/cosmos/staking/v1beta1/params
it is cosmos default i only know cresent has increased it to 28..
and it is a admin param can be changed by gov/ admin